modules/programs/khal.nix

```diff
@@ -147,13 +147,41 @@ let
 in {
   options.programs.khal = {
     enable = mkEnableOption "khal, a CLI calendar application";
+
     locale = mkOption {
       type = lib.types.submodule { options = localeOptions; };
       description = ''
         khal locale settings. 
       '';
       default = { };
     };
+
+    defaultSection = mkOption {
```

Comment:
Maybe just add a `settings`-option? Then people can do

```
settings = {
  default = {
    default_calendar = "calendar";
    timedelta = "5d";
  };
  view = {
    agenda_event_format = "{calendar-color}{cancelled}{start-end-time-style} {title}{repeat-symbol}{reset}";
  };
};
```

Reply:
Sounds like a good plan, I didn't really like the two different-but-the-same sections anyway. And this approach also lets people configure other sections like [highlight_days] and [keybindings] too, so it's a double win.